src/multicellular_stage/MulticellularSpecies.cs

```diff
@@ -28,6 +29,12 @@ public MulticellularSpecies(uint id, string genus, string epithet) : base(id, ge
     [JsonProperty]
     public CellLayout<CellTemplate> Cells { get; private set; } = new();
 
+    /// <summary>
+    ///   The cell layout that this species has in the editor. Is null unless this colony created in the editor.
```

Comment:
I think for eventually we need to have auto-evo also create these layouts as otherwise the player can only really preview their own species which is not the most helpful... So I'd ever so slightly reword this comment.